drools-yaml-rules-core/src/test/java/org/drools/yaml/core/JsonTest.java

```diff
@@ -14,7 +14,7 @@
 
 public class JsonTest {
 
-    private static final String JSON1 =
+    public static final String JSON1 =
```

Comment:
@evacchi Do we need a name for the ruleset

Reply:
it's optional